src/components/ParamsAccordion.tsx

```diff
@@ -0,0 +1,106 @@
+"use client";
+
+import {
+  Accordion,
+  AccordionSummary,
+  AccordionDetails,
+  FormControlLabel,
+  Checkbox,
+  FormControl,
+  FormHelperText,
+  Typography,
+  MenuItem
+} from "@mui/material";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import { InputParams } from "@/models/workflow";
+import FTextField from "@/components/form/FTextField";
+import DragDropUploader from "./DragDropUploader";
+
+interface ParamAccordionGroupProps {
+  groupKey: string;
+  groupParams: InputParams[];
+}
+
+export default function ParamAccordionGroup({
+  groupKey,
+  groupParams
+}: ParamAccordionGroupProps) {
+  return (
+    <Accordion
+      sx={{
+        mb: 2,
+        borderRadius: 2,
+        "&:before": {
+          display: "none"
+        },
+        background: "transparent"
+      }}
+    >
+      <AccordionSummary expandIcon={<ExpandMoreIcon />}>
+        <Typography>
+          {groupKey
```

Comment:
You've written this same code in a few places now, it can probably be turned into a reusable function called `idToTitle()`